feat(genapi): update continue tutorial with latest models (#5354)
scaleway/docs-content: pages/generative-apis/reference-content/adding-ai-to-vscode-using-continue.mdx

- Click **Continue** in the menu on the left.
- In the prompt section, click on **Select model** dropdown, then on **Add Chat model**.
- Select **Scaleway** as provider.
- Select the model you want to use (we recommend `Qwen 2.5 Coder 32b` to get started with chat and autocompletion only).
- Enter your **Scaleway secret key**.
    <Message type="tip">
     To start with, we recommend you use a Scaleway secret key having access to your `default` Scaleway project.
    </Message>

> These actions will edit automatically your `config.yaml` file. To edit it manually, see [Configure Continue through configuration file](#configure-continue-through-configuration-file).

Reviewer: ```suggestion
These actions will automatically edit your `config.yaml` file. To edit it manually, see [Configure Continue through configuration file](#configure-continue-through-configuration-file).
```

<Message type="note">

>   Agent, embeddings, autocomplete models are not yet supported through graphical interface configuration. To enable them, you need to edit the configuration manually, see [Configure Continue through configuration file](#configure-continue-through-configuration-file).

Reviewer: ```suggestion
  Agents, embeddings, autocomplete models are not yet supported through graphical interface configuration. Manually edit the configuration to enable them. See [Configure Continue through configuration file](#configure-continue-through-configuration-file) for more information.
```

</Message>

#### Configure Continue through a configuration file

To link Continue with Scaleway's Generative APIs, you can configure a settings file:

- Open your `config.yaml` settings file:

>   - If your already configured a **Local Assistant**, click on **Local Assistant** then on settings wheel icon to open your existing `config.yaml`

Reviewer: ```suggestion
  - If you have already configured a **Local Assistant**, click **Local Assistant**, then click the **wheel icon** to open your existing `config.yaml`
```

### Going further

> You can add additional parameters to configure your model behaviour by editing `config.yaml`.

Reviewer: ```suggestion
You can add more parameters to configure your model's behavior by editing `config.yaml`.
```
